Patches/Core/ThingDefs_Misc/Apparel_Various.xml

```diff
@@ -258,6 +258,24 @@
 		</value>
 	</Operation>
 
+	<!--<Operation Class="PatchOperationAddModExtension">
+		<xpath>Defs/ThingDef[defName="Apparel_FlakJacket"]</xpath>
+		<value>
+		  <li Class="CombatExtended.PartialArmorExt">
+			  <stats>
+				  <li>
+					Stat def in '<>', value inside, works for both armor rating sharp and blunt
+					<ArmorRating_Sharp>0.25</ArmorRating_Sharp>
+					BodyPartDefs
```

Comment:
This is an odd raw string to include here, is it a human readable label?  If so, it should probably be an XML comment.  Otherwise, it probably can just be skipped.  

Same for the raw string on line 267.

Reply:
> This is an odd raw string to include here, is it a human readable label? If so, it should probably be an XML comment. Otherwise, it probably can just be skipped.
> 
> Same for the raw string on line 267.

It was an example on how the system can be used, I commented it out for when we patch all the apparel later. I couldn't make the while node a comment without removing the comments inside.